client/table.go

```diff
@@ -287,11 +287,11 @@ func (db *DB) CreateTable(desc *structured.TableDescriptor) error {
 		return fmt.Errorf("table \"%s\" already exists", desc.Name)
 	}
 
-	ir, err := db.Inc(keys.DescIDGenerator, 1)
-	if err != nil {
+	if ir, err := db.Inc(keys.DescIDGenerator, 1); err == nil {
+		desc.ID = uint32(ir.ValueInt() - 1)
+	} else {
```

Comment:
unfortunately that won't pass `golint`; if the `if` branch ends in a `return` it is suggested that there be no `else` clause.